testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -650,11 +650,13 @@ def pillar_get(key, minion)
 end
 
 When(/^I enable repositories before installing Salt on this "([^"]*)"$/) do |host|
-  step %(I enable repository "tools_additional_repo" on this "#{host}" without error control)
+  repo = $product == 'Uyuni' ? 'salt3002_repo_repo' : 'tools_additional_repo'
```

Comment:
Silly question, should we just fix sumaform to have as name `tools_additional_repo` instead of `salt3002_repo_repo` when we deploy for Uyuni?

We had the same s--- with Uyuni for Ubuntu, and I fixed it here https://github.com/uyuni-project/sumaform/pull/885

Reply:
They probably have different names because `salt3002_repo_repo` comes from the main.tf and `tools_additional_repo` comes from sumaform.

I am not sure about the reason for that - maybe because `salt3002_repo_repo` won't be there forever as Kostas said?

Please notice that in the test suite code, there are several places where `tools_additional_repo` is skipped for the Uyuni branch.

I don't mind moving this PR to sumaform and having the same name... but we would have to fix all those occurrences, and I am not sure about the consequences.

I would need the help of @juliogonzalez to really understand the difference between `tools_additional_repo` and `salt3002_repo_repo`...

Reply:
Aha, so `salt3002_repo_repo` comes from the main.tf, I though it was on sumaform, then nothing, then we are fine with this PR.